src/ethereum/EthereumNetwork.ts

```diff
@@ -1733,14 +1739,22 @@ export class EthereumNetwork {
         this.ethNeeds.blockHeightLastChecked,
         BLOCKHEIGHT_POLL_MILLISECONDS,
         preUpdateBlockHeight,
-        async () => await this.check('blockheight')
+        async () => {
+          console.log(`nonce`)
```

Comment:
Cleanup console.logs here. Or if you want a real log for the checker, then move it into the check implementation?

Reply:
cleaned up